Library/Formula/gdal.rb

```diff
@@ -13,15 +10,15 @@ class Gdal < Formula
   end
 
   head do
-    url 'https://svn.osgeo.org/gdal/trunk/gdal'
-    depends_on 'doxygen' => :build
+    url "https://svn.osgeo.org/gdal/trunk/gdal"
+    depends_on "doxygen" => :build
   end
 
-  option 'complete', 'Use additional Homebrew libraries to provide more drivers.'
-  option 'enable-opencl', 'Build with OpenCL acceleration.'
-  option 'enable-armadillo', 'Build with Armadillo accelerated TPS transforms.'
-  option 'enable-unsupported', "Allow configure to drag in any library it can find. Invoke this at your own risk."
-  option 'enable-mdb', 'Build with Access MDB driver (requires Java 1.6+ JDK/JRE, from Apple or Oracle).'
+  option "complete", "Use additional Homebrew libraries to provide more drivers."
+  option "enable-opencl", "Build with OpenCL acceleration."
+  option "enable-armadillo", "Build with Armadillo accelerated TPS transforms."
+  option "enable-unsupported", "Allow configure to drag in any library it can find. Invoke this at your own risk."
+  option "enable-mdb", "Build with Access MDB driver (requires Java 1.6+ JDK/JRE, from Apple or Oracle)."
```

Comment:
If we're changing the quoting here could you change these options to be `with-` and use `deprecated_option`? Thanks!